eclipse/dependencies.json

```
@@ -0,0 +1,32 @@
+{
+  "[2019-09,2020-09)": [
+    {
+      "tool": "java",
+      "versionRange": "[8,)"
+    }
+  ],
+  "[2020-09,2022-09)": [
+    {
+      "tool": "java",
+      "versionRange": "[11,)"
+    }
+  ],
+  "[2022-09,2023-09)": [
+    {
+      "tool": "java",
+      "versionRange": "[17,)"
+    }
+  ],
+  "[2023-09,2024-09)": [
+    {
+      "tool": "java",
+      "versionRange": "[21,)"
+    }
+  ],
+  "[2024-09,)": [
+    {
+      "tool": "java",
+      "versionRange": "[22,)"
+    }
+  ]
```

I do not understand where this information is comming from.
I just started Eclipse 2024-09 in IDEasy project using Java21 and it works like a charm.

```suggestion
  "[2023-09,)": [
    {
      "tool": "java",
      "versionRange": "[21,)"
    }
  ]
```

---

I committed your suggsetion.

I found some statements about eclipse 2024-09:

https://www.heise.de/news/Eclipse-IDE-2024-09-Kein-
Support-fuer-Java-7-aber-neue-Texteditorfunktionen-9865067.html

https://www.heise.de/news/Eclipse-IDE-2024-09-Kein-Support-fuer-Java-7-aber-neue-Texteditorfunktionen-9865067.html

Possible I misunderstood something and possible it is vice verse that you need eclipse 2024-09 if you want to use java 23. 

With using 22 instead of 23 was not a good idea. More a typo than reasonable considerations.

Co-authored-by: Jörg Hohwiller <hohwille@users.noreply.github.com>